### PR TITLE
vita3k: Fix compilation errors on the Arch AUR

### PR DESCRIPTION
**Pull request by EXtremeExploit**
#### What does this PR do?
This is the second part of fixing compilation problems in the Arch AUR, turns out that there were more errors than the string literal one, they look to be the same error as before but in different places, they do make sense tho, hopefully this is the last one. If possible i will try to get ownership of it if its a PKGBUILD error, but in the meantime, we should fix these inconsistent strings of the gui

### CODE REVIEW

File: vita3k/gui/src/settings.cpp [hunk 1]
```diff
@@ -422,7 +422,7 @@ void draw_settings(GuiState &gui, HostState &host) {
                         const auto delete_str = is_lang ? lang["delete"] : "This theme will be deleted.";
                         const auto CALC_TEXT = ImGui::CalcTextSize(delete_str.c_str());
                         ImGui::SetCursorPos(ImVec2(POPUP_SIZE.x / 2 - (CALC_TEXT.x / 2.f), POPUP_SIZE.y / 2.f));
-                        ImGui::TextColored(GUI_COLOR_TEXT, delete_str.c_str());
+                        ImGui::TextColored(GUI_COLOR_TEXT, "%s", delete_str.c_str());
```
Comment (Zangetsu38) [inline vita3k/gui/src/settings.cpp:425]: thx, some is already normaly fixed but waiting insidde my setting pr :/

**Comment by Zangetsu38** [comment]
fix format

**Review by Zangetsu38 — APPROVED** [review]
thx sorry for that

**Comment by EXtremeExploit** [comment]
np

**Pull request merged**